modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -504,10 +504,17 @@ public void run() {
         StatusThreadPoolExecutor.getInstance().execute(r);
     }
 
+    public String getKeyStoreDir(String keyUID) {
```

Comment:
Might make more sense to call this `getKeyStorePath`

Reply:
done